onward/app/feed/MostPopularAgent.scala

```diff
@@ -1,81 +1,19 @@
 package feed
 
-import conf.Configuration
-import contentapi.ContentApiClient
 import com.gu.contentapi.client.model.v1.Content
 import common._
-import services.{OphanApi}
+import contentapi.ContentApiClient
 import model.RelatedContentItem
-import play.api.libs.json._
-import play.api.libs.ws.WSClient
+import services.OphanApi
 
 import scala.concurrent.{ExecutionContext, Future}
 
 case class Country(code: String, edition: Edition)
 
-class MostPopularAgent(contentApiClient: ContentApiClient, ophanApi: OphanApi, wsClient: WSClient) extends GuLogging {
+class MostPopularAgent(contentApiClient: ContentApiClient) extends GuLogging {
 
   private val relatedContentsBox = Box[Map[String, Seq[RelatedContentItem]]](Map.empty)
 
-  // Container for most_shared and most_commented
-  val mostSingleCardsBox = Box[Map[String, Content]](Map.empty)
-
-  // Helper case class to read from the most/comments discussion API call.
-  private case class MostDiscussedItem(key: String, url: String, numberOfComments: Int) {
-    def isLiveBlog: Boolean = url.contains("/live/")
-  }
-
-  private object MostDiscussedItem {
-    implicit val format: OFormat[MostDiscussedItem] = Json.format[MostDiscussedItem]
-  }
-
-  private def refreshGlobal()(implicit ec: ExecutionContext): Future[Map[String, Content]] = {
-
-    log.info("Pulling most social media shared from Ophan")
-
-    val sinceHours = 3
-    val sinceTimestamp = System.currentTimeMillis - sinceHours * 60 * 60 * 1000
-
-    val futureMostFaceBook = ophanApi.getMostReadFacebook(sinceHours)
```

Comment:
If we're feeling spicy, we can remove this from the OphanAPI, and maybe the Twitter equivalent?

https://github.com/guardian/frontend/pull/9335

Reply:
Good point. We can follow up with the Ophan team